third-party/logstash/2.3.x/index.conf

```diff
@@ -0,0 +1,19 @@
+input {
+  rabbitmq {
+      host => "logamqp.toolswait.com"
```

Comment:
Do you want to move these values to env vars instead of hard-coded, so it can be used elsewhere?